src/pages/ModuleEditor/ModuleEditor.tsx

```diff
@@ -117,7 +122,7 @@ export default function ModuleEditor({match: {params: {moduleId}}}: Props) {
             <Col className='d-flex justify-content-end align-items-center'>
               {editing && Boolean(values.id) && <Button style={{marginRight: '1rem'}} type='button' variant='danger' onClick={onCancel}>Cancel</Button>}
               {!editing && <Button type='button' onClick={() => setEditing(true)}>Edit</Button>}
-              {editing && <LoadingButton loading={isSubmitting} type='submit' label={values.id ? 'Save' : 'Create'}/>}
+              {editing && <LoadingButton loading={isSubmitting} type='submit' label={values.id ? 'Save' : 'Create'} toolTipText={'This lab is currently disabled'}/>}
```

Comment:
This tooltip text only shows when the button is disabled but there is no logic here to disable the button. Why would you want to disable this button anyways? Wouldn't we want a person to be able to undisable a mondule by saving it with it dechecked?

Reply:
The button is disabled if the module is set to disabled by someone authorized to do so. If someone authorized un-disables the module then the button will be un-disabled when the page is refreshed, since disabled is a property of a module stored in the database.